packages/core-ethereum/src/index.ts

```diff
@@ -140,20 +145,61 @@ export default class HoprEthereum {
     const chain = await createChainWrapper(options?.provider || PROVIDER_DEFAULT_URI, privateKey)
     await chain.waitUntilReady()
 
+    const self = chain.getPublicKey()
+
+    const ownChannels: ChannelEntry[] = []
     const indexer = new Indexer(
       chain.getGenesisBlock(),
       db,
       chain,
       options.maxConfirmations ?? INDEXER_MAX_CONFIRMATIONS,
-      INDEXER_BLOCK_RANGE
+      INDEXER_BLOCK_RANGE,
+      self.toAddress(),
+      (channel: ChannelEntry) => ownChannels.push(channel)
```

Comment:
am not a big fan how this is implemented, a better way to do this would be:

1. wait for indexer to finish syncing
2. load all our channels using `getChannelsOf`
3. filter those without bumpedCommitment & bump commitment
4. using indexer's event listener, wait for new channels and bump commitment

Reply:
obsolete now because implementatio has changed